src/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
```

Comment:
This file was generated by Yeoman generator and I forget to change it, thanks. But I don't know why they made this mix there.